slowfast/config/defaults.py

```diff
@@ -92,6 +93,17 @@
 _C.TEST.CHECKPOINT_TYPE = "pytorch"
 
 
+# ---------------------------------------------------------------------------- #
+# Predicting options
+# ---------------------------------------------------------------------------- #
+_C.PREDICT = CfgNode()
+
+_C.PREDICT.ENABLE = True
+
+_C.PREDICT.LABEL_FILE_PATH = './tmp/kinetics_400_labels.csv'
+
+
+
```

Comment:
Would be good to run the linter :P
`bash linter.sh`
It will help to fix some of the lint issues we have in this PR.